tests/mock_network/node_ageing.rs

```diff
@@ -273,10 +272,6 @@ fn section_churn(
         match churn {
             Churn::Add => {
                 add_node_to_prefix(network, nodes, prefix);
-                unwrap!(nodes.last_mut())
-                    .inner
-                    .set_next_relocation_dst(next_relocation_dst);
```

Comment:
do we need to setup relocation here?

Reply:
See this discussion: https://github.com/maidsafe/routing/pull/1934/files/0a65a186f5434b47b7e2126ccae27b570c38dacb#r353183679